src/Illuminate/Database/Schema/MySqlBuilder.php

```diff
@@ -77,6 +77,10 @@ public function dropAllTables()
             $tables[] = reset($row);
         }
 
+        $tables = array_filter($tables, function ($table) {
+            return str_starts_with($table, $this->connection->getTablePrefix());
```

Comment:
Laravel 8 works with PHP 7.3-8.0.

The [str_starts_with](https://www.php.net/manual/en/function.str-starts-with.php) function was added in PHP 8.0.

Reply:
@andrey-helldar see the discussion above.

Reply:
Or can use `Str::class` because [illuminate/database](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/composer.json#L24) has **illuminate/support** in dependencies.

```suggestion
            return Str::startsWith($table, $this->connection->getTablePrefix());
```